docs/source/enable_cc_lifecycle.md

```diff
@@ -0,0 +1,300 @@
+               "value": {
+               "identities": [
+                  {
+	                 "principal": {
+	           	         "msp_identifier": "Org1ExampleCom",
+	           	         "role": "PEER"
+	                 },
+	                 "principal_classification": "ROLE"
+	              }
+              	],
+              	"rule": {
+                  "n_out_of": {
+			           "n": 1,
+			           "rules": [
+			           	{
+			           		"signed_by": 0
+			           	}
+			           ]
+			       }
+              	},
+              	"version": 0
+              }
+           },
+           "version": "0"
+      }
+   },
+  "org2Policies": {
+      "Endorsement": {
+           "mod_policy": "Admins",
+           "policy": {
+               "type": 1,
+               "value": {
+               "identities": [
+                  {
+	                 "principal": {
+	           	         "msp_identifier": "Org2ExampleCom",
+	           	         "role": "PEER"
+	                 },
+	                 "principal_classification": "ROLE"
+	              }
+              	],
+              	"rule": {
+                  "n_out_of": {
+			           "n": 1,
+			           "rules": [
+			           	{
+			           		"signed_by": 0
+			           	}
+			           ]
+			       }
+              	},
+              	"version": 0
+              }
+           },
+           "version": "0"
+      }
+   },
+   "appPolicies": {
+ 		"Endorsement": {
+			"mod_policy": "Admins",
+			"policy": {
+				"type": 3,
+				"value": {
+					"rule": "MAJORITY",
+					"sub_policy": "Endorsement"
+				}
+			},
+			"version": "0"
+		},
+		"LifecycleEndorsement": {
+			"mod_policy": "Admins",
+			"policy": {
+				"type": 3,
+				"value": {
+					"rule": "MAJORITY",
+					"sub_policy": "Endorsement"
+				}
+			},
+			"version": "0"
+		}
+   },
+   "acls": {
+		"_lifecycle/CommitChaincodeDefinition": {
+			"policy_ref": "/Channel/Application/Writers"
+		},
+		"_lifecycle/QueryChaincodeDefinition": {
+			"policy_ref": "/Channel/Application/Readers"
+		},
+		"_lifecycle/QueryNamespaceDefinitions": {
+			"policy_ref": "/Channel/Application/Readers"
+		}
+   }
+}
+```
+
+## Edit the channel configurations
+
+### System channel updates
+
+Because configuration changes to the system channel only involve parameters
+inside the configuration of the individual ordering service organizations within
+the channel configuration, the relevant organizations can make the edits without
+needing additional signatures.
+
+You will need to export the following variables:
+
+* `CH_NAME`: the name of the system channel being updated.
+* `CORE_PEER_LOCALMSPID`: the MSP ID of the organization proposing the channel
+  update. This will be the MSP of one of the orderer organizations.
+* `ORGNAME`: The name of the organization you are currently updating.
+* `CORE_PEER_TLS_ROOTCERT_FILE`: the absolute path to the TLS cert of your
+  ordering node(s). Need clarification here.
```

Comment:
It is for peer nodes, not orderer nodes. Change the description to something like `the absolute path to the root CA certificate of the organization for which the configuration update will be performed`.